INSTALL.md

```diff
@@ -348,6 +349,26 @@ https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/misc/matrix-
 
 Once you have installed synapse as above, you will need to configure it.
 
+
+## Setting your server_name
+
+Note that you can NOT change the ``server_name`` after the database
+is first created.  So choose your ``server_name`` with care.
+
+You can configure your homeserver to use ``<yourdomain.com>`` as the domain in
+its user-ids, by setting ``server_name`` on the command line:
+
+    python -m synapse.app.homeserver \
+        --server-name <yourdomain.com> \
+        --config-path homeserver.yaml \
+        --generate-config
+    python -m synapse.app.homeserver --config-path homeserver.yaml
```

Comment:
you almost certainly don't want to do this here.

Reply:
Deleted